back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/admin_digest.rb

```diff
@@ -108,30 +108,20 @@ def user_filter_no_invitees(users_scope, _options = {})
 
     def content_worth_sending?(_)
       [
-        statistics.dig(:activities, :new_ideas, :increase),
-        statistics.dig(:activities, :new_initiatives, :increase),
-        statistics.dig(:activities, :new_comments, :increase),
-        statistics.dig(:users, :new_visitors, :increase),
-        statistics.dig(:users, :new_users, :increase),
-        statistics.dig(:users, :active_users, :increase)
+        statistics.dig(:activities, :new_ideas_count),
+        statistics.dig(:activities, :new_comments_count),
+        statistics.dig(:users, :new_users_count)
       ].any?(&:positive?)
     end
 
     def statistics
       @statistics ||= {
         activities: {
-          new_ideas: stat_increase(Idea.pluck(:published_at).compact),
-          new_initiatives: stat_increase(Initiative.pluck(:published_at).compact),
-          new_reactions: stat_increase(Reaction.pluck(:created_at)),
-          new_comments: stat_increase(Comment.pluck(:created_at)),
-          total_ideas: Idea.count,
-          total_initiatives: Initiative.count,
-          total_users: User.count
+          new_ideas_count: stat_increase(Idea.pluck(:published_at).compact),
+          new_comments_count: stat_increase(Comment.pluck(:created_at))
```

Comment:
This is really minor, and happy for you to ignore, but I liked the clarity of using increase in the previous names - maybe `new_ideas_increase` instead of `new_ideas_count`?